tests/unit/location.test.js

```diff
@@ -0,0 +1,15 @@
+const locationAPI = require("../../utils/location");
+
+test("String with city and region when location type is location", async () => {
+  const results = await locationAPI("location");
+  expect(results).toEqual(expect.stringMatching(/\w,\s\w/g));
```

Comment:
i double checked that we wont get a false positive w/ the regex, and you're good to go 🥇